content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -101,32 +101,34 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-### Latency for Istio 1.21
+### Latency for Istio 1.22
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. In the default configuration of Istio 1.21 (i.e. Istio with telemetry v2),
-the two proxies add about 0.182 ms and 0.248 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+proxy. In the default (i.e. ambient mode) configuration of Istio 1.22,
```

Comment:
I'm not sure the default configuration is ambient mode, so maybe this should just say ambient mode, or maybe it's the default ambient mode (just L4?).